geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/CreateAlterDestroyRegionCommandsDUnitTest.java

```diff
@@ -1091,6 +1086,126 @@ public void testDestroyRegionWithSharedConfig() {
     });
   }
 
+
+  final String PR_STRING = " package com.cadrdunit;"
```

Comment:
@kirklund I also thought this code was a bit fancier than maybe it needed to be. But as @davinash mentioned, this is what all of the other deploy jar tests seem to be doing, so I think leaving it consistent with the other test cases is the right thing to do.